client/lib/analytics/hotjar.js

```diff
@@ -45,3 +45,15 @@ export function addHotJarScript() {
 
```

Comment:
Totally feel free to ignore, but there's newer version of this script (`hjsv: 6`) if you'd wanna test updating the script while at it. :-)